### DISCUSSION TITLE
Selecting a specific manufacturer for a BOM component item

### DISCUSSION THREAD

**Discussion by iodize6399** [inventree/InvenTree#3993] [Q&A] [answered]
I have some final assembly products that require components from a specific manufacturer. Various manufacturers and their respective suppliers are already added to that component. Is there a way to select a particular manufacturer for a BOM item?

**Comment by SchrodingersGat** [comment, marked as answer]
@physx2494 this is not possible in the current setup -  BOM items link to [Parts](https://docs.inventree.org/en/latest/part/part/), not [Supplier Parts](https://docs.inventree.org/en/latest/buy/supplier/#supplier-parts)

At the point where you are assigning stock to a Build Order, you can select stock items associated with a particular supplier part, but not encode that information into the Bill of Materials

**Reply by iodize6399** [reply]
Should I go ahead and raise a feature request?

**Reply by SchrodingersGat** [reply]
You can raise a FR, but unless you are willing to implement yourself or sponsor the feature it will not be implemented very quickly. We have a large pile of issues to get through :)

**Comment by eeintech** [comment]
Also don't forget that after exporting the BOM you can manipulate all the supplier and manufacturer parts at your will :smiley:
